.github/workflows/python-package.yml

```diff
@@ -19,7 +19,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest]
-        python-version: ["3.8", "3.9", "3.10"]
+        python-version: ["3.9", "3.10"]
```

Comment:
Why not add 3.11, which we support in our other repos?

Reply:
Good call